packages/@glimmerx/babel-plugin-component-templates/test/fixtures/ember/code.js

```diff
@@ -4,5 +4,5 @@ const unknownValue = null;
 const MaybeComponent = null;
 const maybeModifier = null;
 class MyComponent extends Component {
-  static template = hbs`<h1 {{maybeModifier}}>Hello world {{unknownValue}}<MySubComponent><MaybeComponent /></MySubComponent></h1>`;
```

Comment:
Did `MySubComponent` not exist and now it throws?

Reply:
Yup thats exactly what happens since we are compiling in strict mode.